content/en/blog/news/2024-01-joining-the-cncf-sandbox.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Joining the CNCF Sandbox"
+linkTitle: "Joining the CNCF Sandbox"
+date: 2024-01-24
+---
```

Comment:
Does our current engine supports tags for SEO here?

Reply:
Yes! Pushed a commit with a few tags.